### ISSUE TITLE
Vimeo API ERROR

### ISSUE THREAD

**Issue by greatschism** [Mindinventory/vimeo_video_player#40]
I emailed support and said this is not supported anymore. 

`
'https://player.vimeo.com/video/$vimeoVideoId/config',    
`

```
E/flutter (12076): [ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: type 'Null' is not a subtype of type 'List<dynamic>' in type cast

```

**Comment by prabavgts** [comment]
I am also facing the same issue.Do you have any solution?

**Comment by azjawinski** [comment]
I'm investigating the same problem, after a quick research looks like the direct links to the video are not public anymore and require API auth (that requires a paid Vimeo plan).

So you can either embed their player or pay to use the direct media in your custom player. I may be wrong though 🙈

**Comment by Abdullahfoysal** [comment]
below package is still working:
https://pub.dev/packages/vimeo_player_flutter

**Comment by MrNaifpp** [comment]
i'm facing the same issue.
i fix it by updating flutter version to 3.19.6 the latest version.
hope it helps you

**Comment by abrarmalekji345** [comment]
it seems this issue is reproducible for old Flutter versions, so I am marking it resolved for now, please feel free to reopen it. thanks.
